package.json

```diff
@@ -29,7 +29,7 @@
     "@vue/cli-plugin-eslint": "4.5.13",
     "@vue/cli-plugin-pwa": "4.5.13",
     "@vue/cli-plugin-typescript": "4.5.13",
-    "@vue/cli-service": "4.5.13",
+    "@vue/cli-service": "^4.5.13",
```

Comment:
oh this update was needed ? were there any issues?

Reply:
I guess that got auto updated when I clicked on install npm packages in webstorm